assets/src/settings-page/style.css

```diff
@@ -190,10 +199,13 @@
 	cursor: pointer;
 }
 
-#suppressed-plugins-table .plugin-name,
-#suppressed-plugins-table .column-details p,
-#suppressed-plugins-table .column-details summary {
-	line-height: 30px; /* To match .wp-core-ui select */
+#suppressed-plugins-table summary:hover {
+	color: var(--amp-settings-color-brand);
+}
+
```

Comment:
Is this also necessary?

```suggestion
```

Reply:
I suppose not. I added it when I tried removing the marker.